### DISCUSSION TITLE
Support for Django 3.2 LTS

### DISCUSSION THREAD

**Discussion by FinnGu** [encode/django-rest-framework#7944] [Question & Answer] [answered]
I probably missed it somewhere, but I cannot find an issue where support for the [new version of django](https://docs.djangoproject.com/en/3.2/releases/3.2/), released on 6.4.21, is tracked.

Has anyone successfully upgraded their project yet?

**Comment by carltongibson** [comment, marked as answer]
It all works, pending an update for django-guardian, which I'm not sure is yet compatible. See #7927 and CI runs there.

**Reply by FinnGu** [reply]
Thank you.

**Reply by kucerp21** [reply]
So is Django 3.2. supported or is this merge request needed for the support? I ve tried going through docs, but only Django up to 3.1. is listed

**Reply by FinnGu** [reply]
Hey @kucerp21 ,

I am currently going through the update process from Django 2.1 to 3.2 and have got the server to start with DRF installed. But my old migrations all throw errors, because Django switched the `on_delete` argument of foreign key relations from `CASCADE` to `models.CASCADE`. Don't know if that is helpful to you, but I will keep you updated, once I solve this issue.

**Reply by kucerp21** [reply]
Hey @FinnGu, thanks a lot for the DRF info! 🙂 Personally, I am updating from 3.1 to 3.2, so I think I ll have much less problems. Good luck with your update

**Reply by FinnGu** [reply]
Hey @kucerp21 ,

I have now changed all old migrations and the API works as expected with versions `djangorestframework==3.12.4` and `Django==3.2.3`. I don't think it's related, but currently my logins don't work. I will have to check whether I changed the `SECRET_KEY` or something...
